README.md

```diff
@@ -29,13 +29,17 @@ function CoolPage() {
 
 ## Props
 
-| Prop      | Type   | Description                              | Default |
-| --------- | ------ | ---------------------------------------- | ------- |
-| top       | number | The height where the button gets visible | 20      |
-| color     | string | The arrow color                          | black   |
-| bgColor   | string | Button background color                  | white   |
-| className | string | Custom classname to add your own styling |         |
-| style     | Object | Object to override styling               |         |
+| Prop      | Type    | Description                              | Default |
```

Comment:
What autoformatter/config is this? Seems like some whitespace got added

Reply:
There's no autoformatted here. I had to extend the `Type` column by one characted, because otherwise `boolean` would be sticked to the pipe on the right hand side of it.

Reply:
Ah, my bad